packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -213,7 +214,30 @@ export class RemoteMediaGroup {
   private sendActiveSpeakerMediaRequest(commit: boolean) {
     this.cancelActiveSpeakerMediaRequest(false);
 
-    this.mediaRequestId = this.mediaRequestManager.addRequest(
+    const namedMedia = this.unpinnedRemoteMedia.filter((item) => item?.namedMediaGroup?.value);
```

Comment:
what about `this.pinnedRemoteMedia`? in theory the app can call pin/unpin on any remote media including the named ones. The pinned ones are not part of the active speaker request, but each of them sends their own receive-selected request see RemoteMedia.sendMediaRequest(). Do we need to add namedMediaGroups to the policyInfo there?

Reply:
According to the JMP design: https://confluence-eng-gpk2.cisco.com/conf/display/WMT/JMP+-+Json+Multistream+Protocol , the namedMediaGroups is only defined in "active-speaker" policy.

For "active-speaker" policy,:
- priority (required): the loudest active speakers are mapped to the highest priority requests.  Minimum value is 0, maximum value is 255.  A given active speaker priority MUST NOT be present in multiple different MediaRequests.  Different MediaRequests using active speaker policy MUST have different priorities.
- crossPriorityDuplication (required): when true, the active speakers mapped to a higher priority request are also mapped to lower priority requests.  This parameter MUST have the same value for all requests with "active-speaker" policy.
- crossPolicyDuplication (required): when true, the active speakers mapped to an active-speaker request MAY also be mapped to non-active-speaker requests.
- preferLiveVideo (required): when true, active speakers with live video (e.g. are not audio only or video muted) will be preferred over active speakers without live video.
- namedMediaGroups (optional) = for audio only.  Indicates which named media groups the receiver wants.

Reply:
hmm.... ok, but we need to decide what should happen if the app calls pin() on a remote media object that has a named media group. Maybe we should just not allow that to happen and fail the call to pin()?

Reply:
Right, will throw an error if app calls pin() for named media group